src/about/AboutScreen.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { View,
+          Text,
+          StyleSheet,
+          TouchableOpacity,
+          Linking,
+        } from 'react-native';
+import Logo from './Logo';
+
+export const BRAND_COLOR = 'rgba(66, 163, 109, 1)';
+const styles = StyleSheet.create({
+  initialText: {
+    borderBottomWidth: 0,
+    shadowColor: BRAND_COLOR,
+    shadowOffset: { width: 0, height: 0 },
+    shadowOpacity: 0.5,
+    shadowRadius: 2,
+    elevation: 1,
+    paddingLeft: 3,
+    paddingRight: 3,
+    paddingBottom: 4,
+    marginTop: 3,
+    marginLeft: 5,
+    marginRight: 5,
+  },
+  mainHeader: {
+    marginTop: 40,
+    padding: 5,
+    color: BRAND_COLOR,
+    fontSize: 13,
+  },
+  mainText: {
+    fontSize: 18,
+    textAlign: 'center',
+    fontWeight: 'bold',
+  },
+  subText: {
+    fontSize: 14,
+    textAlign: 'center',
+    marginTop: 5,
+  },
+  screen: {
+    flex: 1,
+    padding: 10,
+  },
+  cardSection: {
+    padding: 3,
+    paddingTop: 5,
+  },
+  insideText: {
+    fontSize: 16,
+    marginBottom: 8,
+  },
+  grayedText: {
+    color: '#BDBDBD',
+    marginLeft: 4,
+    fontSize: 14,
+  },
+  outlineContent: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    marginBottom: 100,
+  },
+  links: {
+    fontWeight: 'bold',
+  },
+  copyrights: {
+    color: '#BDBDBD',
+    textAlign: 'center',
+  },
+});
+class AboutScreen extends React.PureComponent {
+  render() {
+    return (
+      <View style={styles.screen}>
```

Comment:
We should separate these out into separate components so that's its easier to read. As an example:
```
<Logo />
<Title />
<Cards>
    <CardSection>...</CardSection>
...
```